GitUI/Browsing/ICanGetSelectedRevisions.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+using System.DirectoryServices;
+using GitCommands;
+
+namespace GitUI.Browsing
```

Comment:
Could you please explain the choice of the namespace?

Reply:
By analogy with FormBrowse. I wanted to isolate code that is difficult or impossible to place in controls in a separate abstract namespace. Browsing is the main area, Settings is some other area and etc.